drivers/client_access_control/spec/models/grda_warehouse/hud/client_visibility_spec.rb

```diff
@@ -41,7 +41,7 @@
         end
         it 'user can see all clients' do
           expect(GrdaWarehouse::Hud::Client.source.source_visible_to(user).count).to eq(4)
-          expect(GrdaWarehouse::Hud::Client.destination.destination_visible_to(user).count).to eq(4)
+          expect(GrdaWarehouse::Hud::Client.destination.destination_visible_to(user).count).to eq(3)
```

Comment:
Pretty sure this indicates a change in behavior.  I'm getting 4 on the base branch.